src/components/CampDetails.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import StyledChecks from "./shared/StyledChecks";
+import BoxWithLogo from "./shared/BoxWithLogo";
+
+const activities = [
+  {
+    title: "Project-Based Coding",
+    content:
```

Comment:
I didn't add this because we weren't going to use it anymore

Reply:
I'm in a mindset of trying to port the existing signup site over as close to 1:1 as possible